codazen/webDiplomacy
# Feature/fbfw 104 units territories and territory statuses in redux

This PR calls the game/data API route at `?route=game/data&gameID={gameID}&countryID={countryID}` and adds the units, territories, and territory status data into redux so that it is ready to be used. 

### Testing Plan

1. Download the Redux DevTools extension for google chrome https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd
2. Spool up the local environment for Web Dip
3. Create a new game or go to an existing game
4. Click on "Play Beta" to be redirected to the beta app
5. Open the redux devtools extension and verify the correct units, territory, and territory status data is added to the redux store. 

![Screen Shot 2022-04-01 at 1 08 17 PM](https://user-images.githubusercontent.com/93741108/161326755-0d0da47a-eebc-40cb-a6bc-70a21f9a0f04.png)
